Cargo.toml

```diff
@@ -45,40 +45,32 @@ alphanet-instructions = { path = "crates/instructions" }
 alphanet-node = { path = "crates/node" }
 alphanet-precompile = { path = "crates/precompile" }
 
-alloy = { git = "https://github.com/alloy-rs/alloy", rev = "77c1240", features = [
+alloy = { git = "https://github.com/alloy-rs/alloy", rev = "64feb9b", features = [
     "contract",
     "providers",
     "provider-http",
     "signers",
 ] }
-alloy-network = { git = "https://github.com/alloy-rs/alloy", rev = "77c1240" }
-alloy-signer-wallet = { git = "https://github.com/alloy-rs/alloy", rev = "77c1240" }
+alloy-network = { git = "https://github.com/alloy-rs/alloy", rev = "64feb9b" }
+alloy-signer-wallet = { git = "https://github.com/alloy-rs/alloy", rev = "64feb9b" }
 
 # tokio
 tokio = { version = "1.21", default-features = false }
 
 # reth
-reth = { git = "https://github.com/paradigmxyz/reth.git", rev = "a7d8da5", features = ["optimism"] }
-reth-node-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "a7d8da5" }
-reth-node-core = { git = "https://github.com/paradigmxyz/reth.git", rev = "a7d8da5" }
-reth-node-optimism = { git = "https://github.com/paradigmxyz/reth.git", rev = "a7d8da5" }
-reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", rev = "a7d8da5" }
-reth-tracing = { git = "https://github.com/paradigmxyz/reth.git", rev = "a7d8da5" }
+reth = { git = "https://github.com/paradigmxyz/reth.git", rev = "89e55c4", features = ["optimism"] }
+reth-node-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "89e55c4" }
+reth-node-core = { git = "https://github.com/paradigmxyz/reth.git", rev = "89e55c4" }
+reth-node-optimism = { git = "https://github.com/paradigmxyz/reth.git", rev = "89e55c4" }
+reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", rev = "89e55c4" }
+reth-tracing = { git = "https://github.com/paradigmxyz/reth.git", rev = "89e55c4" }
 
 # revm
-revm = { version = "8.0.0", features = ["std", "secp256k1"], default-features = false }
-revm-inspectors = { git = "https://github.com/paradigmxyz/evm-inspectors", rev = "7168ac5" }
-revm-interpreter = { version = "4.0.0", features = ["std"], default-features = false }
-revm-precompile = { version = "6.0.0", features = ["std"], default-features = false }
-revm-primitives = { version = "3.1.0", features = ["std"], default-features = false }
+revm = { version = "9.0.0", features = ["std", "secp256k1", "blst"], default-features = false }
```

Comment:
We probably want to either patch or point to a rev, since the BLS fixes have not been released yet:

https://github.com/bluealloy/revm/compare/v36...main

Reply:
@Rjected great point! the problem with both the patch and pointing to a rev is that we have revm that comes from reth and revm imported with the `blst` feature, if the revs are not the same  `reth::revm::precompile::PrecompileWithAddress` and `revm::precompile::PrecompileWithAddress` no longer match. wdyt, should we wait for a new revm release?

Reply:
we can also merge as is and bump revm when there's a new release, this would unblock the 3074 refactor

Reply:
to be more precise, 3074 refactor is not blocked by these changes, but some of the changes here would be beneficial for it, we can move them to the 3074 refactor work and wait for revm here.

Reply:
hmm, does it still not work since the reth bump here? we don't need to wait, and can just update revm later if so
https://github.com/paradigmxyz/reth/pull/8447/files

Reply:
yep, it works with the latest bump, thx!